packages/blocks/anodot/src/lib/custom-anodot-api-action.ts

```diff
@@ -0,0 +1,133 @@
+import {
+  httpClient,
+  HttpError,
+  HttpHeaders,
+  HttpMethod,
+  HttpRequest,
+  QueryParams,
+} from '@openops/blocks-common';
+import { createAction, Property } from '@openops/blocks-framework';
+import { accountSingleSelectProperty } from './account-property';
+import { anadotAuth } from './anodot-auth-property';
+import {
+  buildUserAccountApiKey,
+  createAnodotAuthHeaders,
+} from './common/anodot-requests-helpers';
+import { authenticateUserWithAnodot } from './common/auth';
+
+export const customAnodotApiAction = createAction({
```

Comment:
community request to support more actions

Reply:
I mean that you can still do, the function is async, you could do all the calls in there, the only thing im not sure how to do is add an extra property, just seems like a shame to have so much code duplication